spec/spec_helper.rb

```diff
@@ -28,6 +28,8 @@
     c.syntax = :expect
   end
 
+  config.alias_it_should_behave_like_to :includes_mixin, 'includes mixin:'
```

Comment:
👎 

`include_examples` and/or `it_behaves_like` are already used throughout our specs and adding our own customized and undocumented alias seems like a bad idea to me. I see where it kinda sorta sounds better given the mixin happy nature of our code but I think people would be much happier knowing exactly what's going on right away (or at least Googleable) with `include_examples "miq ownership"`